index.js

```diff
@@ -39,7 +37,7 @@ module.exports = function (options) {
         return Math.round(wholeNumber / 10) * 10 / multiplier;
     }
 
-    return function (tree) {
+    return function px2rem(tree) {
```

Comment:
name used in PostHTML https://github.com/posthtml/posthtml/blob/master/lib/posthtml.js#L46